client/src/modules/Mentor/pages/Interviews/components/StudentInterview.tsx

```diff
@@ -38,6 +39,11 @@ export function StudentInterview(props: { interview: MentorInterview; template?:
                 </Typography.Link>
               </Typography.Title>
             </Col>
+            <Col>
+              <Typography.Link target="_blank" href={`https://github.com/${student.githubId}`}>
+                <GithubFilled />
+              </Typography.Link>
+            </Col>
```

Comment:
this is not implied by new design. github link will be available on the feedback page. Wondering why one would need to have a link here on the list

Reply:
Don't know, I thought that this could be useful somehow if user requested this